.github/workflows/windows_conditional_compilation.yml

```diff
@@ -62,6 +62,7 @@ jobs:
       CCACHE_DIR: ${{ github.workspace }}\\ccache
       CCACHE_MAXSIZE: 3G
       OPENVINO_REPO: "${{ github.workspace }}\\openvino"
+      ONECORE_TOOLCHAIN: "${{ github.workspace }}\\openvino\\cmake\\toolchains\\onecoreuap.toolchain.cmake"
```

Comment:
looks like you defined this variable in build job, but used  in CC_build job, which is different one.

BTW, maybe the correct place of toolchain usage in Build job, because it uses Ninja, while CC build uses default generator. Could you please update toolchain for Build job instead?